gradle.properties

```diff
@@ -1 +1 @@
-baselineVersion = 0.14.0
+baselineVersion = 0.15.1
```

Comment:
I'd recommend getting rid of this file entirely - the indirection here isn't really that useful!

Reply:
agreed